libtenzir/builtins/operators/http.cpp

```diff
@@ -34,6 +34,7 @@ namespace ssl = caf::net::ssl;
 
 struct http_args {
   located<std::string> url;
+  std::optional<located<bool>> server;
```

Comment:
```suggestion
  std::optional<location> server;
```

Reply:
If we do this then we cannot provide a location if we have a conflict between arguments when explicitly specifying `server=false`.